edge-hub/watchdog/linux/src/main.rs

```diff
@@ -0,0 +1,134 @@
+extern crate nix;
+
+use std::{thread};
+use std::io::Error;
+use std::process::{Command, Stdio, Child};
+use std::sync::{Arc};
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::time::Duration;
+use nix::unistd::Pid;
+use nix::sys::signal::{self, Signal};
+use signal_hook::{iterator::Signals, SIGTERM};
+use tracing::{info, error};
+use tracing_subscriber;
```

Comment:
Let me know if I should order these in a specific way.

Reply:
thanks for asking this. Usually we split into 4 groups divided by a new line. 
- std::
- 3rd party crates (like nix, tracing_
- 1st party crates (like mqtt_broker)
- local (everything crate::...)

Also, it is not needed (anymore) to declare `extern crate`

Reply:
Also, use `cargo fmt` and `cargo clippy` - they will be helpful.